src/components/managementPanel/ManagementPanel.jsx

```diff
@@ -56,142 +88,207 @@ function ManagementPanel() {
-            setUserButtonTile('Unassigned');
-            setTileUsers([]);
-            return console.log('Api problem');
-          }
-          await setTileUsers(response.data.users);
-          setTileButtonTile(`X: ${x}, Y: ${y}`);
-          setSelectedTileData({ id, x, y });
+          handleTileSelected(id);
         },
+        mouseover: () => {
+          setMouseOverTileData(id);
+        }
       }}>
       <Tooltip direction="top">
-        x ={x}, y={y}
+        Y: {x}, Y: {y} <br />
+        {t('managementPanel.assigned')}: {usersCount === 1 ? t('yes') : t('no')} <br />
       </Tooltip>
     </Rectangle>
   ));
 
-  const tilesList =
+  const handleTileSelected = async (tileId) => {
+    loadingUsersState();
+    const selectedTile = tiles.find(x => x.id === tileId);
+
+    setSelectedTileData({
+      id: selectedTile.id,
+      x: selectedTile.x,
+      y: selectedTile.y
+    });
+
+    setSelectedTileId(tileId);
+
+    const response = await getTileUserAssignmentInfo(tileId);
+    if (response.status !== 200) {
+      setTileUsers([]);
+      dispatch(NotificationActions.error(t('unrecognizedProblem')));
+      tilesLoadedState();
+      return;
+    }
+
+    setTileUsers(
+      fulfillTileUsers(response.data.users));
+
+    userSelectedState();
+  };
+
+  const fulfillTileUsers = (users) => {
+    const result = [...users];
+    const assignedUser = result.find(x => x.isAssigned);
+
+    const noneUser =
+    {
+      id: NONE,
+      userName: NONE,
+      isAssigned: assignedUser === undefined
+    };
+
+    result.unshift(noneUser);
+    return result;
+  };
+
+  const tilesDropDown =
     tiles.length > 0
-      ? tiles.map(({ id, x, y, gtfsStopsCount, osmStopsCount }) => (
-        <button
+      ? tiles.map(({ id, x, y }) => (
+        <MenuItem
           key={`tile-dropdown-${id}`}
-          className="dropdown-item"
-          onClick={async () => {
-            const response = await getTileUserAssignmentInfo(id);
-            if (response.status !== 200) {
-              setUserButtonTile('Unassigned');
-              setTileUsers([]);
-              return console.log('Api problem');
-            }
-            await setTileUsers(response.data.users);
-            setTileButtonTile(`X: ${x}, Y: ${y}`);
-            setSelectedTileData({ id, x, y });
-            setUserButtonTile(['Choose User']);
-            setSelectedEditorData({});
-          }}>
-          `{id}; gtfsStopsCount: {gtfsStopsCount}; osmStopsCount: {osmStopsCount}`
-        </button>
+          value={id}>
+          X: {x}, Y: {y}
+        </MenuItem>
+      ))
+      : null;
+
+  const selectedUser =
+    tileUsers.length > 0
+      ? tileUsers.find(x => x.isAssigned).id
+      : '';
+
+  const dropdownUsers =
+    tileUsers.length > 0
+      ? tileUsers.map(({ id, userName, isAssigned }) => (
+        <MenuItem key={id} value={id}>{userName === NONE ? t('managementPanel.noUser') : userName}</MenuItem>
```

Comment:
Could you explain meaning of greater than `>` symbol in this line?

Reply:
It's a closing bracket for MenuItem ;)